tiatoolbox/utils/misc.py

```diff
@@ -567,6 +570,25 @@ def unzip_data(zip_path, save_path, del_zip=True):
         os.remove(zip_path)
 
 
+def save_json(output, output_path):
+    """Convert output to a format supported by json.dumps.
+
+    Args:
+        output (dict): Output dictionary to save.
```

Comment:
please don't use `output` as variable name for input argument. You can call it `in_dict` or `input_dict`.